application/server/server.go

```diff
@@ -276,8 +278,19 @@ func initializeHandler(srv *jrpc2.Server, c *config.Config) handler.Func {
 }
 func initializedHandler(srv *jrpc2.Server) handler.Func {
 	return handler.New(func(ctx context.Context, params lsp.InitializedParams) (any, error) {
+		// Logging these messages only after the client has been initialized.
+		// It has been observed that VSCode changes the log messages format after the LS has been initialized.
```

Comment:
It's because it's after server initialization, the `logMessage` request can be send for logging purposes (see LSP protocol)

Reply:
Thanks, I'll update the comment